doc/api/next_api_changes/behavior/99999-tac.rst

```diff
@@ -0,0 +1,10 @@
+Rename fist arg to subplot_mosaic
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both `.FigureBase.subplot_mosaic`, and `.pyplot.subplot_mosaic` have had the
+first position argument renamed from *layout* to *mosaic*.  This is because we
+are considering to consolidate *constrained_layout* and *tight_layout* keyword
+arguments in the Figure creation functions of `pyplot` into a single *layout*
```

Comment:
```suggestion
arguments in the Figure creation functions of ``pyplot`` into a single *layout*
```

Also rename this file `20150.rst`

Reply:
Went with fixing the xref rather than making it a code literal.